### PR TITLE
Fix Docker image build process

### LINKED ISSUE #285: [BUG] Errors building docker container
I am trying to build a docker container with OpenSearch-Dashboards, but this fails with the following error:
```
   │      Step 23/31 : COPY --chown=1000:0 config/kibana.yml /usr/share/kibana/config/kibana.yml
   │          at makeError (/mnt/build/OpenSearch-Dashboards/node_modules/execa/lib/error.js:59:11)
   │          at handlePromise (/mnt/build/OpenSearch-Dashboards/node_modules/execa/index.js:114:26)
   │          at process._tickCallback (internal/process/next_tick.js:68:7)

error Command failed with exit code 1.
```

The Dockerfile that I think is generated by src/dev/build/tasks/os_packages/docker_generator/templates/Dockerfile seems to reference kibana many times so guessing needs to be updated?

**To Reproduce**
Run the following commands
```
yarn osd bootstrap --force-install --allow-root
yarn build --docker
```

**Expected behavior**
A docker container is built

### PR DESCRIPTION
**Pull request by jgough**
### Description
This fixes the Docker image building process that was still trying to build kibana
 
### Issues Resolved
#285 
 
### Check List
- [ ] New functionality includes testing.
  - [X] All tests pass
- [ ] New functionality has been documented.
  - [ ] New functionality has javadoc added
- [X] Commits are signed per the DCO using --signoff

### CODE REVIEW

File: src/dev/build/tasks/os_packages/docker_generator/templates/Dockerfile
Comment (kavilla) [inline src/dev/build/tasks/os_packages/docker_generator/templates/Dockerfile:19]: Don't need to do for this PR but for these I end up appended [RENAMEME] to the `TODO`'s so that we can track it a little bit easier. So we will end up updating all these artifact links with real links.

**Review by kavilla — DISMISSED** [review]
LGTM! Pulled it down and was able to build docker containers. Thank you!

File: src/dev/build/tasks/os_packages/docker_generator/templates/Dockerfile
Comment (mihirsoni) [inline src/dev/build/tasks/os_packages/docker_generator/templates/Dockerfile:33]: Neat: `OpenSearch Dashboards`

File: src/dev/build/tasks/os_packages/docker_generator/templates/Dockerfile
Comment (mihirsoni) [inline src/dev/build/tasks/os_packages/docker_generator/templates/Dockerfile:40]: Same as above. Use `OpenSearch Dashboards`

**Review by mihirsoni — CHANGES_REQUESTED** [review]
@jgough  Thank you for the PR and changes, few minor comments, otherwise looks good. Where `Kibana` is references in terms of string use `OpenSearch Dashboards` instead of `OpenSearch-Dashboards`

**Review by mihirsoni — APPROVED** [review]
LGTM !! Thanks for the changes @jgough 

I'l wait for @kavilla approval and then merge.

**Review by kavilla — APPROVED** [review]
LGTM!

**Pull request merged**
